src/AutoRest.CSharp/Common/Input/InputTypes/Serialization/TypeSpecInputEnumTypeConverter.cs

```diff
@@ -59,10 +60,9 @@ public static InputEnumType CreateEnumType(ref Utf8JsonReader reader, string? id
             name = name ?? throw new JsonException("Enum must have name");
             // TODO: roll back to throw JSON error when there is linter on the upstream to check enum without @doc
             //description = description ?? throw new JsonException("Enum must have a description");
-            if (description == null)
+            if (description.IsNullOrEmpty())
             {
-                description = "";
-                Console.Error.WriteLine($"[Warn]: Enum '{name}' must have a description");
```

Comment:
We need to give out a message for null or empty description.

Reply:
What message? I copy the behavior from swagger.

Reply:
Why remove the Error log?

Reply:
Added back.